app/models/repository.rb

```diff
@@ -328,23 +328,22 @@ def self.create_from_hash(repo_hash)
     nil
   end
 
-  def check_status(removed = false)
-    Repository.check_status(host_type, full_name, removed)
+  def check_status
+    Repository.check_status(host_type, full_name)
   end
 
-  def self.check_status(host_type, repo_full_name, removed = false)
+  def self.check_status(host_type, repo_full_name)
     domain = RepositoryHost::Base.domain(host_type)
     response = Typhoeus.head("#{domain}/#{repo_full_name}")
 
     if response.response_code == 404
       repo = Repository.includes(:projects).find_by_full_name(repo_full_name)
       if repo
-        status = removed ? nil : "Removed"
-        repo.update_attribute(:status, status) unless repo.private?
+        repo.update_attribute(:status, "Removed") unless repo.private?
         repo.projects.each do |project|
           next unless %w[bower go elm alcatraz julia nimble].include?(project.platform.downcase)
 
-          project.update_attribute(:status, status)
+          project.update_attribute(:status, "Removed")
```

Comment:
I'm wondering why we skip setting removed on the repo, but still set it on the projects, if the repo is private. But, not changed by this PR.